test/qa/lib/login.py

```diff
@@ -51,3 +85,61 @@ def login_as(username, ssh_client: ssh.Ssh = None):
         return ssh_client.exec_command(f"nordvpn login --token {user[1]}")
     else:
         return sh.nordvpn.login("--token", user[1])
+
+
+class SeleniumBrowser():
+    def __init__(self, preferences:list=None):
+        self.options = webdriver.FirefoxOptions()
+        self.options.binary_location = str(sh.which("firefox"))
+        self.options.add_argument('--headless')
+        self.options.add_argument('--no-sandbox')
+
+        if preferences != None:
+            for preference, value in preferences:
```

Comment:
[minor] can't you use `self.browser_set_preferences()` instead?

Reply:
Whoops :smile: seems like I left dead code behind, as `browser_set_preferences()` is not really used anywhere. If I am going to use it one single time, perhaps I should get rid of the method itself? :smile:

Reply:
if you're not using it, you can just remove the method :smile: